docs/src/css/components/_breadcrumb.scss

```
@@ -0,0 +1,189 @@
+
+      font-size: var(--ix-size-sm);
+    }
+  }
+
+  &[data-style='guillemet'] {
+    li:not(:last-child)::after {
+      content: '›';
+
+      color: var(--ix-text-muted);
+
+      font-size: 1.1rem;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)